WordPress/Classes/Services/BlogService.m

```diff
@@ -44,6 +45,9 @@ - (void)flagBlogAsLastUsed:(Blog *)blog
     [defaults setObject:blog.url
                  forKey:LastUsedBlogURLDefaultsKey];
     [defaults synchronize];
+    
+    WP3DTouchShortcutCreator *shortcutCreator = [[WP3DTouchShortcutCreator alloc] init];
```

Comment:
We tend to like to use the `new` shorthand rather than alloc/init. i.e. `[WP3DTouchShortcutCreator new]`